tendrl/notifier/utils/central_store_util.py

```diff
@@ -0,0 +1,57 @@
+from etcd import EtcdException
+from etcd import EtcdKeyNotFound
+from tendrl.commons.objects.alert import Alert
+from tendrl.commons.objects.cluster_alert import ClusterAlert
+from tendrl.commons.objects.node_alert import NodeAlert
+from tendrl.commons.utils import etcd_utils
+
+
+def read(key):
+    result = {}
+    job = {}
+    job = etcd_utils.read(key)
+    if hasattr(job, 'leaves'):
+        for item in job.leaves:
+            if key == item.key:
+                result[item.key.split("/")[-1]] = item.value
+                return result
+            if item.dir is True:
+                result[item.key.split("/")[-1]] = read(item.key)
+            else:
+                result[item.key.split("/")[-1]] = item.value
+    return result
+
+
+def get_alerts():
+    alerts_arr = []
+    alerts = read('/alerting/alerts')
```

Comment:
@shtripat load and load all functions not reading tags alerts, because it is a different json. so i used recursive call here